relay-server/src/actors/store.rs

```diff
@@ -1032,12 +1035,17 @@ struct CheckInKafkaMessage {
 
 #[derive(Debug, Serialize)]
 struct SpanKafkaMessage {
-    /// Raw span data.
+    /// Raw span data. See [`relay_general::protocol::Span`] for schema.
     span: serde_json::Value,
     /// Time at which the span was received by Relay.
     start_time: u64,
     /// The project id for the current span.
     project_id: ProjectId,
+    /// The event id for the current span.
+    ///
+    /// Once spans are truly standalone, this field will be omitted.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    event_id: Option<EventId>,
```

Comment:
Q: is this an option to make the transition to no event_id smoother?

Reply:
That was the idea, yes. `handle_store_envelope` produces kafka messages for span items in an envelope regardless of whether the envelope contains an event.